### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by blrhc**

### CODE REVIEW

File: scoring-periods/0.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+`Intitial Founding Members` (who have already been contacted) will not be able to submit activity summaries as part of Scoring Period #0 as we have already accounted for their previous contributions.
+
+However, if you are not an `Initial Founding Member` but have already made contributions to the project, during the course of Scoring Period #1, simply submit a summary of all of your contributions to the project so far using the instructions [here](../SUBMISSION-GUIDELINES.md) and we will account for this activity with leaderboard points (in the same way as ongoing contributions).
+
+
+#### Processing Note
+
+_Once we have processed all of the summaries submitted in this period and inducted any additional founding members etc., a processing note will be added here with comments and some suggestions for the next scoring period._
+
+_Check back here on `DD/MM/YY` when we expect to have processed all of the qualifying activity summaries._
```
Comment (bwhm) [inline scoring-periods/0.md:16]: I'm not sure we can promise a date here, as we don't know when people will submit.

Comment (bwhm) [reply scoring-periods/0.md:16]: Let's instead say:
- will process within two weeks.
- must submit within 28/02/21 to guarantee grading?

File: scoring-periods/1.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+## `Scoring Period 1`
+### _DD/MM/YY - DD/MM/YY_
```
Comment (bwhm) [inline scoring-periods/1.md:2]: 15/02/21 - 28/02/21

File: scoring-periods/1.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+<br>
+
+_Note: [Scoring Period #0](/0.md) is running at the same time, to recognize existing contributors to the project._
+<br>
+#### Announcement
+We are very happy to announce the inception of our Founding Member program. This initial scoring period will help us to calibrate the program rules and ensure the system works effectively before marketing the program more widely during `Scoring Period #2`.
+
+Existing contributors to the project should also be aware that during this first scoring period, previous contributions to the project can be reported (`Scoring Period #0`). After `Scoring Period #1` has ended, this will no longer be possible.
```
Comment (bwhm) [inline scoring-periods/1.md:10]: I guess it makes more sense to use this deadline.

File: scoring-periods/1.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+We also wanted to recognize our Initial Founding Members, namely:
+- `@username`
```
Comment (bwhm) [inline scoring-periods/1.md:13]: update!

File: scoring-periods/1.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+We would like to thank these community members for their generous ongoing contributions to the project and hope that they will continue to be excellent ambassadors for Joystream as official Founding Members.
+
+
+#### Processing Note
+
+_Once we have processed all of the summaries submitted in this period and inducted any new founding members etc., a processing note will be added here with comments and some suggestions for the next scoring period._
+
+_Check back here on `DD/MM/YY` when we expect to have processed all of the qualifying activity summaries._
```
Comment (bwhm) [inline scoring-periods/1.md:27]: Remember that people won't be able to submit before the end...

File: README.md [hunk 6]
```diff
@@ -1,2 +1,97 @@
...
+      Rules
+    </a>
+    <span> | </span>
+    <a href="https://www.joystream.org/founding-members/">
+      Website
+    </a>
+    <span> | </span>
+    <a href="/scoring-periods/1.md">
+      Period Annoucement
+    </a>
```
Comment (bwhm) [inline README.md:20]: Add:
- Submission Guidelines
- Inducted Founding Members

File: README.md [hunk 7]
```diff
@@ -1,2 +1,97 @@
...
+
+You can also easily submit your regular activity summaries on our website using our [submission form](https://www.joystream.org/founding-members/form/).
+
+## Program Rules
+
+The full rules of the Founding Member Program can be found [here](/RULES.md).
+
+## Submission Guidance
+
+Activity summaries can be submitted using the [form on our website](https://www.joystream.org/founding-members/form), or by sending us a PGP-signed email.
```
Comment (bwhm) [inline README.md:65]:
```
Activity summaries can be submitted using the [form on our website](https://www.joystream.org/founding-members/form), or by sending us an email.
```
They must sign the data, but PGP-encryption is optional.

File: README.md [hunk 8]
```diff
@@ -1,2 +1,97 @@
...
+
+You can find out how to do either of these and the accepted formats for submission [here](/SUBMISSION-GUIDANCE.md). This document also contains a template and exemplar activity summaries, to give you an idea of what we expect submissions to contain.
+
+## Scoring Guidance
+
+We have also added some guidance to highlight what we consider to be some of the highest value contributions towards the growth and success of the project.
+
+Read the non-exhaustive list of contributions and indicative point rewards. [here](/CONTRIBUTIONS.md).
```
Comment (bwhm) [inline README.md:73]: rewards. -> rewards

File: README.md [hunk 9]
```diff
@@ -1,2 +1,97 @@
...
+
+At the end of each scoring period, after internal (Jsgenesis) review and discussion, new Founding Members are inducted. You can read about the previous and current scoring periods below.
+
+_Note: The first scoring period will actually be two scoring periods running at the same time, [Scoring Period #0](/scoring-periods/0.md) and [Scoring Period #1](/scoring-periods/1.md)._
+
+### Current Period
+
+The current scoring period is: [**#1**](/scoring-periods/1.md) <br>(and [_**#0**_](/scoring-periods/0.md), concurrently)
+
+The current scoring period started on `DD/MM/YY` and will end on `DD/MM/YY`.
```
Comment (bwhm) [inline README.md:85]: Dates

File: README.md [hunk 10]
```diff
@@ -1,2 +1,97 @@
...
+
+We have made a public announcement about the current period [here](/scoring-periods/1.md).
+
+### Previous Period
+
+The previous scoring period began on `N/A` and ended on `N/A`.
```
Comment (bwhm) [inline README.md:91]: Fix

File: RULES.md [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+
+Once you have submitted your activity up to and including a certain date, you can no longer submit activity from before this date.
+
+When reporting contributions, the [correct format](/SUBMISSION-GUIDELINES.md) for summary submissions should be observed.
+
+#### Referrals
+
+Any member can earn leaderboard points for referring other contributors to the platform.
+
+Currently, the `referral discount` is 80%, meaning the referrer will earn 0.2 multiplied by the points earned by their referrals.
```
Comment (bwhm) [inline RULES.md:64]: Coefficient probably better, yes :)

File: SUBMISSION-GUIDELINES.md [hunk 12]
```diff
@@ -0,0 +1,79 @@
+## Submission Guidelines
```
Comment (bwhm) [inline SUBMISSION-GUIDELINES.md:1]: I'll clean up this after we merge!
Needs some fixing anyway...

File: CONTRIBUTIONS.md [hunk 13]
```diff
@@ -0,0 +1,88 @@
...
+The scoring system for our leaderboard will be very dynamic and reward a wide range of possible contributions to the success of the project.
+Below are a list of some potential contributions and the approximate number of points you might expect for performing a particular task.
+These numbers are for very rough guidance purposes only, and all points are ultimately awarded based on the judgement of Jsgenesis reviewers.
+
+As we learn more about the evolving ways in which our community can contribute, we will update this list with further tasks and associated scores.
+
+### Example Tasks
+
+Network Integrity:
+- running a validator [indicative score: 500]
```
Comment (bwhm) [inline CONTRIBUTIONS.md:16]: must be over some time period, such as per week.

File: scoring-periods/0.md [hunk 14]
```diff
@@ -0,0 +1,16 @@
+## `Scoring Period 0`
+### _DD/MM/YY - DD/MM/YY_
+<br>
+#### Announcement
+Alongside our first "official" scoring period ([#1](/1.md)) for the Founding Member Program, we will also be running a special one-off scoring period to compensate existing contributors to the project.
+
+`Intitial Founding Members` (who have already been contacted) will not be able to submit activity summaries as part of Scoring Period #0 as we have already accounted for their previous contributions.
```
[the remark this answers is not in the document]
Comment (bwhm) [reply scoring-periods/0.md:7]: I can include that in the next PR

File: CONTRIBUTIONS.md [hunk 15]
```diff
@@ -0,0 +1,88 @@
...
+- webinar/workshop/meetup [indicative score: 1000->10000 per event]
+  - if you would like to organise some sort of event to discuss the platform, this is strongly encouraged
+  - funding may also be available from the Council for these kinds of initiatives
+- providing support in our Telegram channel [indicative score: 25->200 per issue]
+  - providing effective help to fellow participants and especially guiding new joiners through Telegram is a critically important way in which our community can ensure its continued growth and ensure new participants are not lost in the broad scope of the project
+- community proposals [indicative score: 500->5000 per proposal]
+  - if you have a big idea about some project, bounty or other improvement for the community, you can share it with us on GitHub, the platform forum or using the proposals system
+  - depending on whether Jsgenesis and the community likes your idea, it is likely to be eligible for leaderboard points (in addition to funding via the platform)
+
+Accessibility:
```
Comment (bwhm) [inline CONTRIBUTIONS.md:42]: Still not sold on translation: 
- Where will it go?
- Who can grade it?

File: CONTRIBUTIONS.md [hunk 16]
```diff
@@ -0,0 +1,88 @@
...
+- translations [indicative score: 250 per 500 words, depending on quality and subject to review/verification]
+  - translations are very important to make the project accessible to as many people as possible, and we will reward translation of important documents with leaderboard points
+  - note that making original content or tutorials in other languages about the project will also qualify for rewards, though in some cases will warrant less than the equivalent contribution in English
+
+Enriching Testnet:
```
Comment (bwhm) [inline CONTRIBUTIONS.md:47]: May be covered by bounties

File: CONTRIBUTIONS.md [hunk 17]
```diff
@@ -0,0 +1,88 @@
...
+  - participants who draw attention to or write detailed vulnerability reports about issues with our testnet will therefore be rewarded.
+
+Testing:
+- feedback/suggestions on joystream products [indicative score: 500]
+  - users who provide feedback on Joystream applications and offer suggestions for improvements (either through GitHub issues or on the platform forum) will receive leaderboard points
+- detailed testing of joystream products [indicative score: 2500+ depending on scope]
+  - those who want to conduct formal QA testing are likely to be rewarded to an even greater degree than the above
+
+Regular Contributions [indicative score: variable!]:
+- Bounties
```
Comment (bwhm) [inline CONTRIBUTIONS.md:79]: The score will be proportional to the reward, but not sure what the reward to score should be...

File: CONTRIBUTIONS.md [hunk 18]
```diff
@@ -0,0 +1,88 @@
...
+  - The completion of official bounties can be included in your regular Founding Member summary and these will usually result in points being awarded.
+- KPIs
+  - If you can provide evidence that you played a significant role in the Council achieving a specific KPI, you can also include this in your regular summary.
+- Roles
+  - Ongoing participation in platform roles will also qualify for leaderboard points.
+
+Referrals:
+- Referring other people to the project who go on to submit leaderboard summaries will also qualify for rewards (they don't have to ultimately become Founding Members, just earn points).
+  - At the moment the referral discount is 80% meaning you will earn 0.2x the points earned by your referrals.
```
Comment (bwhm) [inline CONTRIBUTIONS.md:88]: again - discount

**Review by bwhm — CHANGES_REQUESTED** [review]
Getting close to the finish line!
Would be good if you could make the changes ASAP, then I'll add in my stuff!

File: avatars/README.md [hunk 19]
```diff
@@ -0,0 +1,41 @@
...
+Your avatar is unique to you, and is yours forever.
+You will even be able to use it on the Joystream platform on mainnet.
+
+We will also display your avatar on our leaderboards on `joystream.org` and in our social media channels when recognizing your contributions to the project.
+
+This repo holds a number of different variants of each avatar, which can be compared below.
+
+#### Primary Avatar
+
+These can be found in the `/primary-avatar/` folder.
```
[the remark this answers is not in the document]
Comment (blrhc) [reply avatars/README.md:18]: Should have been fixed in commit above.

File: avatars/README.md [hunk 20]
```diff
@@ -0,0 +1,41 @@
+# Founding Member Avatars
+
+### What Are Avatars For?
+
+As part of the process of being formally admitted and inducted into the Founding Member program, you will be asked to select a custom illustrated avatar for the duration of your membership of the program.
+
+You can use your avatar anywhere you would like, though you are particuarly encouraged to use the avatar on Joystream products, such as in Pioneer and in our media consumption application.
+
+Your avatar is unique to you, and is yours forever.
```
[the remark this answers is not in the document]
Comment (blrhc) [reply avatars/README.md:9]: Noted, but not sure this is the right place to explain this.

File: avatars/README.md [hunk 21]
```diff
@@ -0,0 +1,41 @@
...
+You will even be able to use it on the Joystream platform on mainnet.
+
+We will also display your avatar on our leaderboards on `joystream.org` and in our social media channels when recognizing your contributions to the project.
+
+This repo holds a number of different variants of each avatar, which can be compared below.
```
[the remark this answers is not in the document]
Comment (blrhc) [reply avatars/README.md:14]: Yes, will definitely add SVGs later

File: avatars/README.md [hunk 22]
```diff
@@ -0,0 +1,41 @@
...
+
+#### Primary Avatar
+
+These can be found in the `/primary-avatar/` folder.
+
+<img width="100" src="/primary-avatar/1.png">
```
[the remark this answers is not in the document]
Comment (blrhc) [reply avatars/README.md]: Could do...
but when we add people to the program we will only offer them "free" avatars anyway.
I like having an established "id" for each avatar and the name change obscures this a bit.

File: scoring-periods/0.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (blrhc) [reply scoring-periods/0.md:7]: > I can include that in the next PR

Yes please do, I think you have a better grasp of the activity, though I will definitely review.

File: scoring-periods/1.md (hunk 5 above)
Comment (blrhc) [reply scoring-periods/1.md:27]: Oh, I thought they could submit at all times?
Didn't we agree that would make it easier to feel an immediate sense of achievement if summaries could be submitted whenever and processed as soon as possible?
The logistics of that do seem more complex though.

**Comment by bwhm** [comment]
May include some smaller cleanups with my follow up, but looks good!

**Pull request merged**